eng/pipelines/templates/jobs/release.yml

```diff
@@ -85,9 +87,46 @@ jobs:
             ServiceEndpointUrl: 'https://api.esrp.microsoft.com'
             MainPublisher: 'ESRPRELPACMANTEST'
             ProductState: latest
+
+        - if eq('${{ parameters.IncludeNative }}', true):
+          - task: EsrpRelease@9
+            displayName: Publish native platform packages to npmjs.com
+            inputs:
+              ConnectedServiceName: 'Azure SDK PME Managed Identity'
+              ClientId: '5f81938c-2544-4f1f-9251-dd9de5b8a81b'
+              DomainTenantId: '975f013f-7f24-47e8-a7d3-abc4752bf346'
+              UseManagedIdentity: true
+              KeyVaultName: 'kv-azuresdk-codesign'
+              SignCertName: 'azure-sdk-esrp-release-certificate'
+              Intent: 'PackageDistribution'
+              ContentType: 'npm'
+              FolderLocation: $(Pipeline.Workspace)/drop/${{ parameters.ServerName }}-native/platform
```

Comment:
Can you please make sure this pattern is getting signed and the signing verification includes the `-native` variation.

Reply:
If the structure remains as such:

<img width="771" height="549" alt="image" src="https://github.com/user-attachments/assets/eb336768-8658-4d16-adfe-6418ef807055" />

It should get signed. However, we should look into why the pdbs are in there.